parmed/openmm/topsystem.py

```diff
@@ -47,6 +47,10 @@ def load_topology(topology, system=None, xyz=None, box=None):
         information unless ``box`` (below) is also specified
     box : array of 6 floats
         Unit cell dimensions
+    unique_atom_Types : bool, default=True
```

Comment:
`unique_atom_Types` -> `unique_atom_types` (capitalization).

I'll admit to being confused by the behavior here.  I read `unique_atom_types` as "assign all atom types to be unique", when what this really means is that atom types should be condensed into as few atom types as possible.

I think this parameter is better named `condense_atom_types=True` to be more reflective of what it really does.